experimenter/experimenter/nimbus-ui/src/components/LinkNav/index.tsx

```diff
@@ -64,6 +70,20 @@ export const LinkNav = ({
         >
           {children}
         </Button>
+      ) : isExternalRoute ? (
+        <Button
+          {...{ title }}
+          variant="link"
+          href={to}
+          className={classNames(
+            textColor,
+            "d-flex font-weight-semibold m-0 p-0 b-0 align-items-center",
+          )}
+          data-testid={testid}
+          onClick={onClick}
+        >
+          {children}
+        </Button>
```

Comment:
To reduce code duplication, could you add `href={isExternalRoute ? to : ""}` to the above block? This prop appears to be the only difference between the two Buttons.